brightness-mode-explainer.md

```diff
@@ -0,0 +1,131 @@
+# Design Document: the need to control screen brightness
+
+_As discussed in the [DAS WG 2021-11-17](https://www.w3.org/2021/11/17-dap-minutes.html) meeting about Screen Brightness and the Screen Wake Lock API, the idea of this document is to make it easier for the meeting attendees to work on an explainer than if this had been committed to GitHub directly. Once this is done the goal is to move the contents to a proper explainer on GitHub._
+
+Previously:
+- https://github.com/WICG/proposals/issues/17
+- https://github.com/w3c/screen-wake-lock/issues/129
+- https://github.com/w3c/devicesensors-wg/issues/51
+- https://www.w3.org/2021/11/17-dap-minutes.html
```

Comment:
We should probably move all this to the end of this document and move the Introduction's text here . This stuff is  only relevant for us WG members, but not for anyone else.

Reply:
Done. I've dropped the italicized paragraph since we're already on GitHub and moved the links to a new section at the end of the document.